### PR TITLE
Added French translation

### PR DESCRIPTION
**Pull request by Bindou01**
I tried to translate most of it as best I could, but there may be some small things that could be improved. Especially the names of the different views. For now, I preferred to keep the name "artistic", but I think we can do better.

### CODE REVIEW

File: language/resource.language.fr_fr/strings.po [hunk 1]
```diff
@@ -0,0 +1,2284 @@
...
+
+#: /1080i/Includes_Labels.xml
+msgctxt "#31499"
+msgid "In-Production"
+msgstr "En production"
+
+#: /1080i/Includes_Search.xml
+msgctxt "#31500"
+msgid "There were no [LOWERCASE]$VAR[Label_Setting_SearchStyle][/LOWERCASE] results found for"
+msgstr "Aucun résultat trouvé pour [MINUSCULE]$VAR[Label_Setting_SearchStyle][/MINUSCULE]"
```
Comment (jurialmunkey) [inline language/resource.language.fr_fr/strings.po]: These are formatting tags. You shouldn't translate `[LOWERCASE]`

File: language/resource.language.fr_fr/strings.po [hunk 2]
```diff
@@ -0,0 +1,2284 @@
...
+msgstr "Défilement automatique des boîtes de texte"
+
+#: /1080i/SkinSettings.xml
+msgctxt "#31247"
+msgid "Colour highlight"
+msgstr "Surbrillance de la couleur"
+
+msgctxt "#31248"
+msgid "Building includes"
+msgstr "Construction des inclusions"
```
Comment (jurialmunkey) [inline language/resource.language.fr_fr/strings.po]: Shouldn't translate the word "includes" as this is a technical term which refers to specific skin engine files (the `Includes.xml` files)

**Review by jurialmunkey — CHANGES_REQUESTED** [review]
A couple of very minor changes required but otherwise looks great!

File: language/resource.language.fr_fr/strings.po (hunk 2 above)
Comment (Bindou01) [reply language/resource.language.fr_fr/strings.po]: Hi Jurial, It's fixed.

I may make some small adjustments as I see the translations in context, in use.

File: language/resource.language.fr_fr/strings.po (hunk 1 above)
Comment (Bindou01) [reply language/resource.language.fr_fr/strings.po]: I don't know if it's relevant to translate this part [Label_Setting_SearchStyle] ?

Comment (jurialmunkey) [reply language/resource.language.fr_fr/strings.po]: No that one should be left as is too. 

That's a skin variable. It was originally used to change whether it said "Search" or "Discover" however I just noticed I've accidentally reused the variable name so I need to fix that up -- but effectively that $VAR[] part will switch between "Search" or "Discover" depending on the window which is open.

**Pull request merged**